apps/aecore/src/aec_db.erl

```diff
@@ -158,6 +182,33 @@ get_header(Hash) ->
            Header
        end).
 
+find_block(Hash) ->
+    ?t(case mnesia:read(aec_blocks, Hash) of
+           [#aec_blocks{txs = Txs}] ->
+               [#aec_headers{value = Header}] = mnesia:read(aec_headers, Hash),
+               {value, aec_blocks:from_header_and_txs(Header, Txs)};
+           [] -> none
+       end).
+
+find_header(Hash) ->
+    case ?t(mnesia:read(aec_headers, Hash)) of
+        [#aec_headers{value = Header}] -> {value, Header};
+        [] -> none
+    end.
+
+find_chain_node(Hash) ->
+    case ?t(mnesia:read(aec_headers, Hash)) of
+        [H] -> header_to_node(H);
+        [] -> none
+    end.
+
+find_chain_node_successors(Hash) ->
+    Objects = ?t(mnesia:index_read(aec_headers, Hash, prev_hash)),
```

Comment:
I see that mistyping the index name here (e.g. `prev_hash3`) causes `make test` to fail. Good.